explorer/Explorer.tsx

```diff
@@ -483,21 +483,26 @@ export class Explorer
     }
 
     @action.bound private onResize() {
+        const oldIsNarrow = this.isNarrow
         this.isNarrow = isNarrow()
-        this.grapherBounds = this.getGrapherBounds() || this.grapherBounds
+        this.updateGrapherBounds()
+
+        // If we changed between narrow and wide mode, we need to wait for CSS changes to kick in
+        // to properly calculate the new grapher bounds
+        if (this.isNarrow !== oldIsNarrow)
+            window.setTimeout(() => this.updateGrapherBounds(), 0)
```

Comment:
Awesome that you figured this out!

The fix feels a bit fragile – we can't guarantee that the component is rendered after `setTimeout` right? 

But it's definitely better than what we have, so fine with me to ship it like this for now.

Reply:
Ah yeah, you're right. We can be sure that the browser has applied any already-lined-up CSS changes by then, but we cannot be sure that React actually does that.

I quickly looked at `componentDidUpdate`, which would be the way to go here, but sadly we don't have access to `prev.isNarrow` there since it's neither a prop nor React state :/